var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -145,6 +145,7 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
             when="@:7",  # req in CP2K v8+
             description="Use CUBLAS for general matrix operations in DBCSR",
         )
+    variant("rocm", description="building with rocm", default=False)
```

Comment:
CP2K is a ROCm package, so this variant should already be available from the base class. Am I missing something?

Reply:
You're right. I will remove the line.